src/flows/pipes/Notification/Threshold.tsx

```diff
@@ -67,7 +67,8 @@ const Threshold: FC<Props> = ({readOnly}) => {
             type: deadmanType,
             deadmanCheckValue: '5s',
             deadmanStopValue: '90s',
-            field: threshold.field || '_value',
+            field: threshold.field || 'Select a field',
+            fieldType: '',
```

Comment:
what's fieldType used for?

Reply:
good question.
ErrorThreshold and Threshold are the same type signature, except for `fieldType`.
`fieldType` is currently utilized in ErrorThreshold only, not yet in Threshold.
The next PR (where the implicit data schema will be fetched), will allow `fieldType` to be added to Threshold as well.

The use of `fieldType` in ErrorThreshold is by defining it as 'number' or 'not-number'. Thresholds should probably have the same options...but undecided. Either way, will make it consistent for all Thresholds (be it Error or not).

Reply:
Gotcha.